sdks/java/harness/src/test/java/org/apache/beam/fn/harness/FnApiDoFnRunnerTest.java

```diff
@@ -658,17 +500,4 @@ private ByteString encode(String ... values) throws IOException {
     }
     return out.toByteString();
   }
-
-  @Test
-  public void testRegistration() {
-    for (Registrar registrar :
-        ServiceLoader.load(Registrar.class)) {
-      if (registrar instanceof FnApiDoFnRunner.Registrar) {
```

Comment:
This test should stick around but we should ensure that the ParDo URN is registered.